nx-X11/programs/Xserver/hw/nxagent/Window.c

```diff
@@ -2674,6 +2686,18 @@ void nxagentDisconnectWindow(void * p0, XID x1, void * p2)
   }
   #endif
 
+  if ((nxagentRealWindowProp) && (!nxagentWindowTopLevel(pWin)))
+  {
+    Atom prop = MakeAtom("NX_REAL_WINDOW", strlen("NX_REAL_WINDOW"), True);
```

Comment:
Hmm... Hmmm... Hmmm... I prefer not to. As the rest of the code does it similarly.